seed/migrations/0171_update_bae_assets.py

```diff
@@ -0,0 +1,68 @@
+# Generated by Django 3.2.12 on 2022-06-29 14:29
+# this is identical to migration 0163 but BAE was updated so need to change (to include bae v0.1.9)
+
+from django.db import migrations
+
+from seed.lib.xml_mapping.mapper import get_BAE_mappings
+
+
+def update_bae_fields(apps, schema_editor):
+    """create a default BuildingSync column mapping preset for each organization"""
+    Organization = apps.get_model("orgs", "Organization")
+
+    # profile number for 'BuildingSync Default' profile is 1
+    prof_type = 1
+
+    # 'Audit Template Building Id' name matches the automatically generated field name from
```

Comment:
remove this commented text, all the way to line 34.

Reply:
done!